package ch.raspamo;

import ch.raspamo.Exception.ExerciseExeption;

public interface Callable {
    public void call() throws ExerciseExeption;
}
